### ISSUE TITLE
Open source serveo server

### ISSUE THREAD

**Issue by paperbenni** [milio48/serveo#14]
Although unmaintained, serveo is still one of the most popular tunneling solutions and the self hosted server still works great. The problem is that it'll eventually break and with the source not being available to the public there's not much we could do about that. 
If you published the source, the community could maybe maintain it.

**Comment by dannyb648** [comment]
Yeah I second this. Really wanted to base some infrastructure around Serveo in June, but the system being down made me look for alternatives. If it was open source I'd consider it

**Comment by holyCowMp3** [comment]
Look at the [sish](https://github.com/antoniomika/sish) - i used it as part of my infrastructure - it is a pretty stable and customisable.
